scaler/queue_pinger.go

```diff
@@ -65,7 +66,7 @@ func (q *queuePinger) count() int {
 func (q *queuePinger) requestCounts() error {
 	log.Printf("queuePinger.requestCounts")
 	endpointsCl := q.k8sCl.CoreV1().Endpoints(q.ns)
-	endpoints, err := endpointsCl.Get(q.svcName, metav1.GetOptions{})
+	endpoints, err := endpointsCl.Get(context.TODO(), q.svcName, metav1.GetOptions{})
```

Comment:
@khaosdoctor can you add a `context.Context` argument to the function, and pass it to this function?

Reply:
Done on 1983310 however I'm creating the context on the pinger file (https://github.com/khaosdoctor/http-add-on/blob/improve-conditions/scaler/queue_pinger.go#L49)